docs/0001-nixos-render-docs-Output-GFM-admonition.patch

```diff
@@ -44,5 +44,5 @@ index 4ff0bc3095c3..e151325338ca 100644
  [
  multiline
 -- 
-2.46.0
+2.45.2
```

Comment:
Not sure I understand what this version change does

Reply:
@MattSturgeon and I woul have had different git versions, as I re-generated the patch so it would be 1/2

Reply:
Ah, alright, I rarely look that closely at git patches. That makes sense.